packages/flutter_driver/lib/src/common/find.dart

```diff
@@ -119,17 +119,17 @@ class WaitUntilNoTransientCallbacks extends Command {
   String get kind => 'waitUntilNoTransientCallbacks';
 }
 
-/// A Flutter Driver command that waits until the frame is synced.
-class WaitUntilNoPendingFrame extends Command {
-  /// Creates a command that waits until there's no pending frame scheduled.
-  const WaitUntilNoPendingFrame({ Duration timeout }) : super(timeout: timeout);
+/// A Flutter Driver command that waits until the Flutter app is idle.
+class WaitUntilIdle extends Command {
```

Comment:
@tvolkert will this need to be announced as a breaking change?

Reply:
I added this API a couple of days ago (which shouldn't be used anywhere yet...), and kept the waitUntilNoTransientCallbacks API. LMK if this is a breaking change.

Reply:
Since that API is so young, maybe revert that change so nobody starts depending on it until we have agreement what API we actually want to add?